### PR TITLE
Updated iris sample code in README.

### PR DESCRIPTION
**Pull request by naitoh**
Updated iris sample code in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,11 +32,51 @@ iris.each do |record|
...
+# => ["Iris-setosa", "Iris-setosa", .. , "Iris-versicolor", ..
+```
+
+Here is an example to access iris dataset by `Table#fetch_values`:
+
+```ruby
+require "datasets"
+
+iris = Datasets::Iris.new
+p iris.to_table.fetch_values(:sepal_length, :sepal_width, :petal_length, :petal_width).transpose
```
Comment (kou) [inline README.md]: How about defining `iris_table = iris.to_table` local variable?
Calling `to_table` for each `fetch_values` is costful.

File: README.md [hunk 2]
```diff
@@ -32,11 +32,51 @@ iris.each do |record|
...
+# => [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 ```
 
+Here is an example to access iris dataset by `Table#to_h`:
+
+```ruby
+require "datasets"
+
+iris = Datasets::Iris.new
+iris_table = iris.to_table.to_h
```
Comment (kou) [inline README.md]: `iris_hash` instead of `iris_table` will be a better name.

File: README.md (hunk 1 above)
Comment (naitoh) [reply README.md]: Thank you very much.
Fixed.

File: README.md (hunk 2 above)
Comment (naitoh) [reply README.md]: Thank you very much.
Changed.

File: README.md [hunk 3]
```diff
@@ -32,11 +32,51 @@ iris.each do |record|
...
-  # [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
-  # [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 end
+# => [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
+# => [4.9, 3.0, 1.4, 0.2, "Iris-setosa"]
+  :
+# => [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 ```
 
+Here is an example to access iris dataset by `Table#to_h`:
```
[the remark this answers is not in the document]
Comment (naitoh) [reply README.md]: I put together the example of Iris and added the URL.
And, the URL of CIFAR was also added.

The name of the data set matched the notation of the Web page.
Thanks!

**Comment by naitoh** [comment]
The review content was fixed.
Thank you.

**Comment by kou** [comment]
Thanks!

**Pull request merged**